string name
float64 linear_value
float64 angular_value